Gemfile.lock

```diff
@@ -74,6 +74,13 @@ GEM
     docile (1.3.2)
     erubi (1.9.0)
     execjs (2.7.0)
+    factory_bot (6.1.0)
```

Comment:
Should we get rid of these gems or do we want to spend a little time understanding them?